adding-to-and-updating-the-cart.md

```diff
@@ -118,28 +119,28 @@ You can add multiple purchasables to the cart in an update cart form. You supply
     {{ csrfInput() }}
 
     {% for variant in product.variants %}
-        <input type="hidden" name="purchasables[{{loop.index}}][id]" value="{{ variant.id }}">
-        <input type="hidden" name="purchasables[{{loop.index}}][qty]" value="1">
-        <input type="hidden" name="purchasables[{{loop.index}}][note]" value="1">
+        <input type="hidden" name="purchasables[{{ loop.index }}][id]" value="{{ variant.id }}">
+        <input type="hidden" name="purchasables[{{ loop.index }}][qty]" value="1">
+        <input type="hidden" name="purchasables[{{ loop.index }}][note]" value="1">
     {% endfor %}
 
     <input type="submit" value="Add all variants to cart">
 </form>
 ```
 
-While using multi-add the same rules apply for updating a quantity vs adding to cart, based on the uniquessness of the options `signature` and `purchasableId`.
+While using multi-add the same rules apply for updating a quantity vs. adding to cart, based on the uniquessness of the options `signature` and `purchasableId`.
```

Comment:
The attribute on the line item is called `optionsSignature` which then hash of the `options` attribute on the line item.